internal/storage/rocksdb/store.go

```diff
@@ -412,23 +520,47 @@ func (rdb *rocksDB) SaveChanges(changes []*serverpb.ChangeRecord) (uint64, error
 type iter struct {
 	iterOpts storage.IterationOptions
 	rdbIter  *gorocksdb.Iterator
+	ttlCF    bool
 }
 
 func (rdb *rocksDB) newIter(iterOpts storage.IterationOptions) *iter {
 	readOpts := rdb.opts.readOpts
 	it := rdb.db.NewIterator(readOpts)
-
-	if sk, prsnt := iterOpts.StartKey(); prsnt {
+	if sk, present := iterOpts.StartKey(); present {
 		it.Seek(sk)
 	} else {
 		it.SeekToFirst()
 	}
-	return &iter{iterOpts, it}
+	return &iter{iterOpts, it, false}
+}
+
+func (rdb *rocksDB) newIterCF(iterOpts storage.IterationOptions) *iter {
```

Comment:
Please check if the start key logic duplication between `newIter` and `newIterCF` can be reduced.